CONTRIBUTING.md

```diff
@@ -1,49 +1,53 @@
 
-### Organization
+## Organization
 
-To repeat from the ReadMe, the guidelines are divided into these sections:
+The guidelines are currently divided into these sections:
 
-* [Style Guide (Introduction)](Style-Guide/Introduction.md)
-  * [Code Layout and Formatting](Style-Guide/Code-Layout-and-Formatting.md)
-  * [Function Structure](Style-Guide/Function-Structure.md)
-  * [Documentation and Comments](Style-Guide/Documentation-and-Comments.md)
-  * [Readability](Style-Guide/Readability.md)
-  * [Naming Conventions](Style-Guide/Naming-Conventions.md)
-* [Best Practices (Introduction)](Best-Practices/Introduction.md)
-  * [Naming Conventions](Best-Practices/Naming-Conventions.md)
-  * [Building Reusable Tools](Best-Practices/Building-Reusable-Tools.md)
-  * [Output and Formatting](Best-Practices/Output-and-Formatting.md)
-  * [Error Handling](Best-Practices/Error-Handling.md)
-  * [Performance](Best-Practices/Performance.md)
-  * [Security](Best-Practices/Security.md)
-  * [Language, Interop and .Net](Best-Practices/Language-Interop-and-.Net.md)
-  * [Metadata, Versioning, and Packaging](Best-Practices/Metadata-Versioning-and-Packaging.md)
+- [Style Guide (Introduction)](Style-Guide/Introduction.md)
+  - [Code Layout and Formatting](Style-Guide/Code-Layout-and-Formatting.md)
+  - [Function Structure](Style-Guide/Function-Structure.md)
+  - [Documentation and Comments](Style-Guide/Documentation-and-Comments.md)
+  - [Readability](Style-Guide/Readability.md)
+  - [Naming Conventions](Style-Guide/Naming-Conventions.md)
+- [Best Practices (Introduction)](Best-Practices/Introduction.md)
+  - [Naming Conventions](Best-Practices/Naming-Conventions.md)
+  - [Building Reusable Tools](Best-Practices/Building-Reusable-Tools.md)
+  - [Output and Formatting](Best-Practices/Output-and-Formatting.md)
+  - [Error Handling](Best-Practices/Error-Handling.md)
+  - [Performance](Best-Practices/Performance.md)
+  - [Security](Best-Practices/Security.md)
+  - [Language, Interop and .Net](Best-Practices/Language-Interop-and-.Net.md)
+  - [Metadata, Versioning, and Packaging](Best-Practices/Metadata-Versioning-and-Packaging.md)
 
-Markdown documents on GitHub support linking within a document, but only to headers, so when editing, in addition to keeping practices and guidelines in the documents where they make sense, please use headlines for each guideline, and lower level headlines for rationale, examples, counter examples, and exceptions.
+GitHub supports linking within a markdown document, but only to headings. When editing, in addition to keeping practices and guidelines in the document where they make sense, please include a heading for each guideline, and level down heading for rationale, examples, counterexamples, and exceptions.
```

Comment:
Having made the earlier comment, we should probably amend this section to say "GitHub and GitBooks support" ...